### PR TITLE
Adding option to include datadog plugin

### PR DESCRIPTION
**Pull request by peripateticus**
- Adds option to include datadog plugin during app scaffold

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +76,10 @@ The author of the NPM package we're initializing. Attempts to resolve the user's
 
 The type of template we're initializing. Defaults to `vue`.
 
+`-g, --datadog`
```
Comment (HeavyMedl) [inline README.md]: `-g` is slightly confusing to me. Maybe `-dd`?

File: src/bin/main.mts [hunk 2]
```diff
@@ -16,7 +17,7 @@ const program = new Command();
 program
   .name('create-package')
   .description('An NPM initializer that scaffolds new NPM packages')
-  .version('2.5.0');
+  .version(packageJson.version);
```
Comment (HeavyMedl) [inline src/bin/main.mts:20]: <3

File: src/bin/main.mts [hunk 3]
```diff
@@ -39,12 +40,13 @@ program
     'package template',
     Defaults.PACKAGE_TEMPLATE,
   )
+  .option('--datadog', 'Configure application to be monitored by DataDog', Defaults.INCLUDE_DATADOG)
```
Comment (HeavyMedl) [inline src/bin/main.mts]: Oh, looks like `-g` isn't actually an available option

File: src/types/index.mts [hunk 4]
```diff
@@ -11,5 +12,7 @@ export interface Answer {
   packageTemplate: string;
   namespacedDir: string;
   packageOwnerTeamId: string;
+  packageUsesDataDog: true;
```
Comment (HeavyMedl) [inline src/types/index.mts]: Should this be `boolean`?

File: src/util.mts [hunk 5]
```diff
@@ -18,6 +18,7 @@ export const Defaults = {
   PACKAGE_AUTHOR: 'REI',
   PACKAGE_NAMESPACE: true,
   PACKAGE_TEMPLATE: 'VUE', // Should map to TemplateTypes
+  INCLUDE_DATADOG: true,
```
Comment (HeavyMedl) [inline src/util.mts]: My instinct here is that this value should be `false` by default. Using DD should always be opt-in.

File: src/types/index.mts (hunk 4 above)
Comment (peripateticus) [reply src/types/index.mts]: Yep! Nice catch!

File: src/bin/main.mts (hunk 3 above)
Comment (peripateticus) [reply src/bin/main.mts]: Oh right, yeah I must have thought the same thing and just removed that. Should we do `-dd, --datadog`?

File: src/types/index.mts (hunk 4 above)
Comment (peripateticus) [reply src/types/index.mts]: Even TS can't stop me from being dumb! 🤣

File: src/util.mts (hunk 5 above)
Comment (peripateticus) [reply src/util.mts]: Yep, that makes sense, everything else around DD is opt-in.

File: src/bin/main.mts (hunk 3 above)
Comment (HeavyMedl) [reply src/bin/main.mts]: I think just `--datadog` is good.

File: src/types/index.mts (hunk 4 above)
Comment (HeavyMedl) [reply src/types/index.mts]: Honestly, I'm surprised it didn't yell at you here. `true` is a value, not a type, right?

**Review by HeavyMedl — APPROVED** [review]

**Pull request merged**